app/src/main/java/org/gnucash/android/ui/common/BaseDrawerActivity.java

```diff
@@ -108,15 +98,18 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(getContentView());
+        inflateView();
+        mDrawerLayout = findViewById(R.id.drawer_layout);
```

Comment:
mDrawerLayout = mBinding.drawLayout;

Reply:
mBinding is not available here, since it only exists in subclasses. (Same in BaseReportFragment).